plugins/aws/iam/iamUserWithoutPermissions.js

```diff
@@ -0,0 +1,121 @@
+    recommended_action: 'Modify IAM user and attach new permissions',
+    apis: ['IAM:listUsers', 'IAM:listUserPolicies', 'IAM:listAttachedUserPolicies', 'IAM:getPolicyVersion' ,'IAM:listGroupsForUser',
+        'IAM:listGroups', 'IAM:listGroupPolicies', 'IAM:listAttachedGroupPolicies'],
+
+    run: function(cache, settings, callback) {
+        var results = [];
+        var source = {};
+        
+        var region = helpers.defaultRegion(settings);
+
+        var listUsers = helpers.addSource(cache, source,
+            ['iam', 'listUsers', region]);
+
+        if (!listUsers) return callback(null, results, source);
+
+        if (listUsers.err || !listUsers.data) {
+            helpers.addResult(results, 3,
+                'Unable to query for user IAM policy status: ' + helpers.addError(listUsers));
+            return callback(null, results, source);
+        }
+
+        if (!listUsers.data.length) {
+            helpers.addResult(results, 0, 'No user accounts found');
+            return callback(null, results, source);
+        }
+        
+        async.each(listUsers.data, function(user, cb){
+            if (!user.UserName) return cb();
+
+            var listAttachedUserPolicies = helpers.addSource(cache, source,
+                ['iam', 'listAttachedUserPolicies', region, user.UserName]);
+
+            var listUserPolicies = helpers.addSource(cache, source,
+                ['iam', 'listUserPolicies', region, user.UserName]);
+
+            var listGroupsForUser = helpers.addSource(cache, source,
+                ['iam', 'listGroupsForUser', region, user.UserName]);
+
+            if (!listAttachedUserPolicies) return cb();
+            if (!listUserPolicies) return cb();
+            if (!listGroupsForUser) return cb();
+
+
+            if (listAttachedUserPolicies.err) {
+                helpers.addResult(results, 3,
+                    'Unable to query for IAM attached policy for user: ' + user.UserName + ': ' + helpers.addError(listAttachedUserPolicies), 'global', user.Arn);
+                return cb();
+            }
+
+            if (listUserPolicies.err) {
+                helpers.addResult(results, 3,
+                    'Unable to query for IAM user policy for user: ' + user.UserName + ': ' + helpers.addError(listUserPolicies), 'global', user.Arn);
+                return cb();
+            }
+
+            if (listGroupsForUser.err) {
+                helpers.addResult(results, 3,
+                    'Unable to query for IAM groups attached to user: ' + user.UserName + ': ' + helpers.addError(listGroupsForUser), 'global', user.Arn);
+                return cb();
+            }
+
+
+            if (!listAttachedUserPolicies.data || !listUserPolicies.data || !listGroupsForUser.data) {
+                helpers.addResult(results, 3, 'Unable to query policies for user: ' +
+                    user.UserName + ': no data returned', 'global', user.Arn);
+                return cb();
```

Comment:
should have separate msg for each one of them, might result in confusion if list groups for user dont return anything